package-lock.json

```diff
@@ -330,6 +330,12 @@
             }
           }
         },
+        "conventional-changelog-preset-loader": {
```

Comment:
Not sure why this file is modified.

Reply:
It was changed when I ran `npm test` which seemed odd but I decided to commit it as it was probably needed?